src/components/Playlist/Playlist.js

```diff
@@ -31,12 +31,18 @@ export default function Playlist({
   )
 }
 
-const PlaylistGrid = styled.section``
+const PlaylistGrid = styled.section`
+  display: grid;
+`
 
 const WarmupWrapper = styled.div`
   display: grid;
 `
 
-const IntervalsTWrapper = styled.div``
+const IntervalsTWrapper = styled.div`
+  display: grid;
+`
```

Comment:
If all your wrappers just do `display: grid`, you could reuse the same wrapper and don't need multiple ones.